docker/README.md

```diff
@@ -76,21 +76,19 @@ and enter the Provider's URL.
 
 Go to https://localhost/api_keys, generate key and secret. Then run:
 
-    docker exec -it openqa_data /scripts/client-conf set -l KEY SECRET
+    docker exec -it openqa_data /scripts/client-conf set -l -t webui_nginx_1 KEY SECRET
     # Where KEY is your openQA instance key
     # and SECRET is your openQA instance secret
 
 ## Run the Worker container
 
-    # Fedora
-    docker run -d -h openqa_worker_1 --name openqa_worker_1 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged fedoraqa/openqa_worker
+    docker run -d -h openqa_worker_1 --name openqa_worker_1 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged openqa_worker
 
 Check whether the worker connected in the WebUI's administration interface.
 
 To add more workers, increase number that is used in hostname and container name, so to add worker 2 use:
 
-    # Fedora
-    docker run -d -h openqa_worker_2 --name openqa_worker_2 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged fedoraqa/openqa_worker
+    docker run -d -h openqa_worker_2 --name openqa_worker_2 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged openqa_worker
```

Comment:
sorry but again this is something completely unrelated. You add a tag when starting the webUI  container and you change the run instructions for the worker container. "fedoraqa/openqa_worker" is a valid name for the container on docker hub. You can propose to change that for a good reason but please not in the same git commit.

Reply:
I remove this part